fastplotlib/graphics/selectors/_linear.py

```diff
@@ -99,6 +99,9 @@ def __init__(
             name of linear selector
 
         """
+        self._fill_color = None
```

Comment:
Set the initial values from the user args?

Reply:
Linear selector is weird: it uses an inner and outer line so it doesn't conform to the vertices, edges, and fill schema...

Reply:
only the inner line color should be set, rename the kwargs `edge_color`?

Reply:
yeah, I think it makes it more uniform